libraries/core_libs/consensus/include/dag/sortition_params_manager.hpp

```diff
@@ -77,6 +77,8 @@ class SortitionParamsManager {
   int32_t getChange(uint64_t period, uint16_t efficiency) const;
   void cleanup(uint64_t current_period);
 
+  const uint64_t k_threshold_testnet_hard_fork_period = 50000;
```

Comment:
I know it is only for testnet but we vote, that constant values naming should be kConstatValue